package/src/frontend/browser/Agent.js

```diff
@@ -86,6 +86,43 @@ export const Agent = function(userAgent) {
      */
     matchesFacebookInAppBrowser: function() {
       return userAgent.match(/FBAN/) && userAgent.match(/FBAV/);
+    },
+
+    matchesDesktopChrome: function() {
+      return this.matchesChrome() && !this.matchesMobilePlatform() && minVersion(/Chrome\/(\d+)/i, 25);
+    },
+
+    matchesDesktopFirefox: function() {
+      return this.matchesFirefox() && !this.matchesMobilePlatform() && minVersion(/Firefox\/(\d+)/i, 20);
+    },
+    
+    matchesDesktopEdge: function() {
+      return this.matchesEdge() && !this.matchesMobilePlatform() && minVersion(/Edg\/(\d+)/i, 20);
+    },
```

Comment:
Let's make `minVersion` a parameter of the function to be more useful and not require a min version by default. This will also make the tests more explicit since they will not depend on the hard coded values in these functions. We can instead move these min versions to the `isBrowserSupported` function of the scrolled editor.

Reply:
You mean like this?
```
matchesDesktopChrome: function(minVersion) {
      return this.matchesChrome() && !this.matchesMobilePlatform() && matchesMinVersion(/Chrome\/(\d+)/i, minVersion);
    },
```

Or there could be one other option to be separate functions for separate browsers like:
```
matchesDesktopChrome: function(minVersion) {
      return this.matchesChrome() && !this.matchesMobilePlatform() && matchesMinChromeVersion(/Chrome\/(\d+)/i, minVersion);
    },
```

Reply:
Think about it from the perspective of the client - the code that uses `browser.agent`. If I call `agent.matchesDesktopChrome()`, I'd expect it to return true for any version of Chrome on Desktop. I'd pass the min version in an options object: `agent.matchesDesktopChrome({minVersion: 25})` instead of `agent.matchesDesktopChrome(25)`. The latter could be misread as requiring exactly version 25. Also this makes it easier to add options in the future if we have to.